src/main/kotlin/calculator/Separates.kt

```diff
@@ -0,0 +1,33 @@
+package calculator
+
+data class Separates(val value: List<String>){
+    fun toIntList(): List<Int> {
```

Comment:
Int형 List로 변환해주고 있네요. 👍 
```suggestion
    fun toIntList(): PositiveNumbers {
```
지금도 잘 구해주셨지만, PositiveNumber를 활용해보면 어떨까요?